lib/openactive/helpers/json_ld.rb

```diff
@@ -33,6 +33,21 @@ def prepare_data_for_serialization(obj, parent = nil, schema: false)
               ]
           end
 
+          # push @'s to the start
+          data = data.sort_by do |k, _v|
+            if k == '@context'
```

Comment:
How about a case statement as an alternative to if/elsif here?

Reply:
Alternatively I was thinking:
```
if k.start_with('@)
  ["@context", "@type", "@id"].index(k) || 3
else
  4
end
```

Reply:
Ha - Nice, that would also work. I think the more explicit if/elsif or case statement is clearer to understand though.